rfd/0-job-worker.md

```diff
@@ -25,7 +25,7 @@ The worker library defines a simple interface with methods to start, stop, query
 
 ##### cgroups
 
-The library implementation will be configured at instantiation with limits that can be parsed into values for `cpu.max`, `memory.max` and `io.max`. These values will originate from runtime flags provided to the server with defaults that do not implement limits. All processes will be started in a new cgroup regardless of the values of these limits. Note that for expediency, it is assumed that cgroups v2 are in use on the system running the service.
+The library implementation will be configured at instantiation with limits that can be parsed into values for `cpu.max`, `memory.max` and `io.max`. These values will originate from runtime flags provided to the server with useful defaults. All processes will be started in a new cgroup regardless of the values of these limits. Note that for expediency, it is assumed that cgroups v2 are in use on the system running the service.
```

Comment:
How do you pick the devices for io.max?

Reply:
i've just been playing around with that... i can get the devices by looking in `/sys/block` (ignoring `loop` devices) and then get the device numbers from `/proc/partitions`. It doesn't feel very clean, but it would work.

Reply:
That works for me.